pkg/kv/dynamodb/store.go

```diff
@@ -113,9 +113,13 @@ func (d *Driver) Open(ctx context.Context, kvParams kvparams.Config) (kv.Store,
 		}
 	})
 
-	err = setupKeyValueDatabase(ctx, svc, params)
-	if err != nil {
-		return nil, fmt.Errorf("%w: %s", kv.ErrSetupFailed, err)
+	// Create table if not exists
+	success, _ := isTableExist(ctx, svc, params.TableName)
```

Comment:
Can you please documentation explaining why we need isExist and not satisfied with the response of the create table